src/pages/[language]/blog/index.vue

```diff
@@ -1,127 +1,10 @@
-<template>
-  <main class="page-blog">
-    <page-header
-      heading="byline"
-      :byline="data.page.title"
-      :headline="data.page.subtitle"
-      :image="data.page.headerIllustration"
-    />
-
-    <section class="page-blog-container grid">
-      <h2 class="sr-only">
-        {{ $t('blog_overview') }}
-      </h2>
-
-      <text-block class="page-blog__text">
-        <p class="testimonial">
-          {{ data.page.description }}
-        </p>
-      </text-block>
-
-      <ul class="page-blog__posts">
-        <li
-          v-for="blogPost in data.page.pinnedPosts"
-          :key="blogPost.slug"
-        >
-          <blog-list-item
-            large
-            pinned
-            :item="blogPost"
-          />
-        </li>
-        <li
-          v-for="blogPost in data.items.filter(post => post.published)"
-          :key="blogPost.slug"
-        >
-          <blog-list-item
-            large
-            :item="blogPost"
-          />
-        </li>
-      </ul>
-    </section>
-
-    <section class="page-blog__pivots grid">
-      <pivot-list
-        v-if="data.page.pivots && data.page.pivots.length"
-        :pivots="data.page.pivots"
-        :can-have-border-top="false"
-        :pivot-narrow="true"
-      />
-
-      <div class="page-blog__scroll-to">
-        <scroll-to direction="up" />
-      </div>
-    </section>
-  </main>
-</template>
-
 <script setup>
-  import query from './index.query.graphql?raw';
-
-  const { params } = useRoute();
-  const { data } = await useFetchContent({
-    query,
-    variables: {
-      locale: params.language,
+definePageMeta({
+  middleware: [
+    // eslint-disable-next-line no-unused-vars
+    function (to, from) {
+      return navigateTo({ name: 'language-blog-page-page', params: { page: 1 }})
```

Comment:
Thoughts on putting this redirect on 'server level', in our case Netlify redirects?